inspirehep/dojson/common/base.py

```diff
@@ -105,9 +121,7 @@ def control_number2marc(self, key, value):
 @journals.over('spires_sysnos', '^970..')
 @hepnames.over('spires_sysnos', '^970..')
 @jobs.over('spires_sysnos', '^970..')
-@utils.ignore_value
 def spires_sysnos(self, key, value):
-    """Old SPIRES number and new_recid from 970."""
     external_system_numbers = self.get('external_system_numbers', [])
```

Comment:
Should we use this opportunity to unify all schemas by using `external_system_identifiers` like in hep instead?